web/src/Components/Btn.elm

```diff
@@ -35,9 +36,13 @@ btn props inner =
                 Small ->
                     [ class "btn-outer--small" ]
 
+                VSmall ->
+                    [ vary BtnSmall True ]
+
                 Click msg ->
                     [ onClick msg ]
 
+                -- TODO: Btn is not safely disabled. Can be removed with Dev Tools
```

Comment:
That's fine - should have a check for appropriate conditions somewhere. Remember that a sophisticated user will just figure out how to issue the request manually if they want to.

Security and UX are two diff considerations, and we shouldn't handle security when thinking about UX stuff - that should be way deeper.